docs-utils/troubleshooting-structure-adjustment.js

```diff
@@ -11,12 +11,21 @@ function moveArticleTagsToFrontmatter(filePath) {
     if (fileContent.includes('**Tags:**')) {
         const tags = fileContent.split('**Tags:**')[1].split('\n')[0]
```

Comment:
```suggestion
        const tags = fileContent.split('**Tags:**')[1].split('\n')[0].trim()
```

Reply:
There is whitespace before and after some tags, like in docs/en/troubleshooting/Integrations/why-dont-marketplace-orders-integrate-with-my-store.md

I think adding a trim here would fix it